ui/app/pages/mobile-sync/mobile-sync-intro-popup.js

```diff
@@ -0,0 +1,56 @@
+import React, { useContext } from 'react'
```

Comment:
It seemed reasonable to put this new popup in mobile-sync folder, as we're promoting mobile.  If desired, I can create a `mobile` directory.